tools/gui/tools_gui.py

```diff
@@ -0,0 +1,135 @@
+'''
+Python ver 2.7.15
+This is a GUI for any tools that require one, such as the plotter that graphs
+csv files and a config maker to customize graphs.
+
+NOTE: There is a test file under tools/test/test_csv to test out the plotter.
+'''
+import Tkinter as tk
+import ttk, tkFileDialog
+#Importing the plotter backend. 
+import sys, os
+sys.path.append(os.path.join(os.path.dirname(__file__),"plotter"))
+import plotter.plotter_backend as plotter_tool
+
+class multiToolGUI:
+  '''
+  A class used to represent the GUI for the plotter (and future tools).
+
+  Methods
+  -------
+  select_file()
+    Gets the user chosen filename and filepath and creates widgets for the
+    user to choose their graph labels in a drop-down menu
+  submit_graph_options()
+    Retrieves user-selected graph labels and plots a graph
+  setIcon()
+    Sets the GUI favicon
+  '''
+
+  def __init__(self):
+    self.main_window = tk.Tk() #Creates the application window
+    self.main_window.title("MST Multirotor Multitool")
```

Comment:
Configurable things such as the window title should have an associated constant
```suggestion
    self.main_window.title(WINDOW_TITLE)
```